packages/vscode-extension/src/controls/sampleGallery/sampleCard.tsx

```diff
@@ -15,103 +15,70 @@ import {
 } from "../../telemetry/extTelemetryEvents";
 import { Commands } from "../Commands";
 import { Setting, Watch } from "../resources";
-import { SampleCardProps } from "./ISamples";
+import { SampleProps } from "./ISamples";
 
-export default class SampleCard extends React.Component<SampleCardProps, any> {
-  constructor(props: SampleCardProps) {
+export default class SampleCard extends React.Component<SampleProps, unknown> {
+  constructor(props: SampleProps) {
     super(props);
   }
 
   render() {
+    const sample = this.props.sample;
     return (
       <div
-        className={`sample-card box${this.props.order}`}
+        className={`sample-card`}
         tabIndex={0}
-        onClick={this.onSampleCard}
+        onClick={this.onSampleCardClicked}
         onKeyDown={(e) => {
           if (e.key === "Enter") {
-            this.onSampleCard();
+            this.onSampleCardClicked();
           }
         }}
       >
-        {this.props.suggested && (
+        {sample.suggested && (
           <div className="triangle">
             <FontIcon iconName="FavoriteStar" className="star"></FontIcon>
           </div>
         )}
-        <label
```

Comment:
These labels are for accessibility. Any reason to remove them?

Reply:
Sorry I thought they are legacy codes. I will try to add them back.